src/sql/migrations.rs

```diff
@@ -671,6 +671,10 @@ CREATE INDEX smtp_messageid ON imap(rfc724_mid);
         )
         .await?;
     }
+    if dbversion < 97 {
+        sql.execute_migration("DELETE FROM config WHERE keyname = 'attach_selfavatar'", 97)
+            .await?;
+    }
```

Comment:
i think, we have quite some config that are "left over" and we cannot be sure what is in the config table anyway (eg. even if this is exectued, `attach_selfavatar` may be present if ppl use an older version after updating).

migrations have maintenance costs and always some risk, even if low in this case, i personally would skip the migration completely.